src/routes/formsgSiteCreation.ts

```diff
@@ -157,9 +166,83 @@ export class FormsgRouter {
     await mailer.sendMail(email, subject, html)
   }
 
+  cloneSiteToEfs: RequestHandler<
+    never,
+    Record<string, never>,
+    { data: { submissionId: string } },
+    never,
+    { submission: DecryptedContent }
+  > = async (req, res) => {
+    // 1. Extract arguments
+    const { submissionId } = req.body.data
+    const { responses } = res.locals.submission
+    // NOTE: This is validated by formsg to be of domain `@open.gov.sg`;
+    // hence, not revalidating here
+    const requesterEmail = getField(responses, "Email") as string
```

Comment:
should we just allow whitelisted emails?

our admin email perhaps?

Reply:
To be clear, you want to stricten from allowing `@open.gov.sg` to only `admin@open.gov.sg`? I don't see too much of an issue here but what's the extra benefit ah cos this only prevents against internal attacks

Reply:
just suggesting [least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) principle. Not too much issue, since this is all internal, will let it up to you to decide

Reply:
^not doing, the experience of having prod ops enter their own email then receive a rejection email saying it must be admin@isomer.gov.sg seems less than ideal given that this is all internal users anyway (validation of `@open.gov.sg` domain)